core/src/main/scala/org/broadinstitute/dsde/rawls/user/UserService.scala

```diff
@@ -264,7 +264,10 @@ class UserService(protected val userInfo: UserInfo, val dataSource: SlickDataSou
    * */
   def unregisterBillingProjectWithOwnerInfo(projectName: RawlsBillingProjectName, ownerInfo: Map[String, String]): Future[PerRequestMessage] = {
     val ownerUserInfo = UserInfo(RawlsUserEmail(ownerInfo("newOwnerEmail")), OAuth2BearerToken(ownerInfo("newOwnerToken")), 3600, RawlsUserSubjectId("0"))
-    unregisterBillingProjectWithUserInfo(projectName, ownerUserInfo)
+    for {
+      _ <- deleteGoogleProjectIfChild(projectName, ownerUserInfo)
```

Comment:
`deleteGoogleProjectIfChild` will actually delete the Google project in Google-land which will cause problems for gpalloc